app/hypnos/src/main.c

```diff
@@ -26,13 +28,15 @@ void bt_thread(void);
 #define PRIORITY 7
 
 K_THREAD_DEFINE(bt_id, STACKSIZE, bt_thread, NULL, NULL, NULL,
-                PRIORITY, 0, 0);
+			PRIORITY, 0, 0);
```

Comment:
This makes indentation wrong. I've got Emacs configured to use Linux C style and that puts PRIORITY just below bt_id.